README.md

```diff
@@ -89,6 +89,50 @@ class ExampleController < ActionController::Base
 end
 ```
 
+## Configuration
+
+Gate give possibility to create global, inherited configuration and separated setup in each one schema definition.
+
+### Inheriting base setup
+
+In order to configure the library according to whole application, create file under `config/initializers/gate.rb`. There is a possibility of creating base configuration, definining shared, custom predicates and a lot of more useful things. The full list of supported configuration options is consistent with available options in [dry-validation](https://dry-rb.org/gems/dry-validation/) gem.
```

Comment:
`s/defininig/defining`
`s/file/a file`
`s/creating base/creating a base`
`s/lot of more/lot more`
`s/to whole/to the whole`